supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/Delocalization.scala

```diff
@@ -47,45 +50,72 @@ trait Delocalization {
-      // Redirect the result to a file that can be used in the next action to delocalize the extracted files / directories
-      s" > $outputFile"
+      // Create the directory where the fofn will be written
+      s"mkdir -p $$(dirname $outputFile) && " +
+      s"cd $containerCallRoot && " +
+      s"${womOutputRuntimeExtractor.command} > $outputFile"
     )
 
     ActionBuilder
-      .withImage("stedolan/jq@sha256:a61ed0bca213081b64be94c5e1b402ea58bc549f457c2682a86704dd55231e09")
+      .withImage(womOutputRuntimeExtractor.dockerImage.getOrElse(cloudSdkImage))
       .setCommands(commands.asJava)
       .withMounts(mounts)
       .setEntrypoint("/bin/bash")
       // Saves us some time if something else fails before we get to run this action
       .withFlags(List(ActionFlag.DisableImagePrefetch))
   }
+  
+  private def delocalizeRuntimeOutputsScript(fofnPath: String, workflowRoot: Path, cloudCallRoot: Path) = {
+    val gsutilCommand: String => String = { flag =>
+      s"""gsutil -m $flag cp -r $$line "${cloudCallRoot.pathAsString.ensureSlashed}$$gcs_path""""
+    }
+    
+    def sedStripPrefix(prefix: String) = s"""sed -e "s/^${prefix.sedEscaped}//""""
+    
+    // See RuntimeOutputMapping.prefixFilters for more info on why this is needed
+    val prefixFilters = RuntimeOutputMapping
+      .prefixFilters(workflowRoot)
+      .map(sedStripPrefix)
+      .mkString(" | ")
 
-  private def delocalizeOutputJsonFilesAction(cloudCallRoot: Path, inputFile: String, workflowRoot: String, mounts: List[Mount]): Action = {
-    val sedStripSlashPrefix = "s/^\\///"
-    val gsutilCommand: String => String = flag => s"""gsutil -m $flag cp -r % ${cloudCallRoot.pathAsString.ensureSlashed}$$(echo % | sed -e "$sedStripSlashPrefix")"""
-    val command =
-        // Read the file containing files to delocalize
-        s"cat $inputFile 2>/dev/null" +
-        /*
-         * Pipe the result to xargs and execute a gsutil cp command, appending the path to the cloudCallRoot
-         * Use a subshell so that we can strip the potential leading slash from the local path and avoid double slashes in the cloud path
-         * It also seems that the gsutil fails without sub shelling
-         * We can't use the gsutil -I flag here because it would lose the directory structure once it gets copied to the bucket
-         * sh -c 'gsutil cp % $(echo % | sed -e "s/^\///")'
-         */
-        s""" | xargs -I % sh -c '${recoverRequesterPaysError(cloudCallRoot)(gsutilCommand)}'"""
-
-    ActionBuilder.cloudSdkShellAction(command)(mounts)
+    /*
+     * Delocalize all the files returned by the runtime output extractor
+     */
+    s"""
+       |#! /bin/bash
+       |
+       |set -x
+       |
+       |if [ -f $fofnPath ]; then
+       |  while IFS= read line
+       |  do
+       |    gcs_path=$$(echo $$line | $prefixFilters)
+       |    ${recoverRequesterPaysError(cloudCallRoot)(gsutilCommand)}
+       |  done  <$fofnPath
+       |fi
+     """.stripMargin
```

Comment:
I believe this IntelliJ copypasta leaves some extra whitespace at the beginning of the file, so the shebang won't be on the first line. Either start your string with `s"""|#!/bin/bash`, or change this to `.stripMargin.trim`.